### DISCUSSION TITLE
KAKFA bridge to use MTLS connection to bootstrp

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-bridge#746] [General]
HI the example manifest  has  bootstrapServers: my-cluster-kafka-bootstrap:9092  ,   in my case 9092 as well is MTLS , is there any instruction as to how I can implement bridge to connect to MTLS bootstrap?

**Comment by scholzj** [comment]
It is both in docs and in the examples. So please check there.

**Reply by sreejesh123** [reply]
Thank you , will check again
